components/nimbus/src/client/mod.rs

```diff
@@ -3,7 +3,7 @@
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
 mod fs_client;
-mod http_client;
+pub mod http_client;
```

Comment:
nit: Hmm does this need to be public?

Reply:
without this `pub` all of the `http_client` stuff is inaccessible: 
<img width="592" alt="image" src="https://user-images.githubusercontent.com/43795363/154339280-8a572a01-7bd2-4883-b414-be4256483750.png">

I can make it `pub(crate)` though